bench/db_bench.ml

```diff
@@ -221,9 +240,20 @@ module Lmdb = struct
 
   let read_random r = print_results (read r random) nb_entries
 
-  let read_seq r = print_results (read r sorted) nb_entries
-
-  let read_seq_rev r = print_results (read r (List.rev sorted)) nb_entries
+  let read_seq r =
+    let read (txn, ddb) () =
+      opencursor txn ddb >>= fun cursor ->
+      cursor_first cursor >>= fun () ->
+      cursor_iter
+        ~f:(fun (k, v) ->
+          ignore (Bigstring.to_string k);
```

Comment:
you probably don't want to call `Bigstring.to_string` at all here, as this is copying bytes and allocating a string in the heap which is not what we want to measure -- I think it's just fine to ignore `k` and `v`.

Reply:
don't we want to add the time to convert the big string to the overall time ?

Reply:
I guess it really depends on what we are trying to measure :-)